lesson_07/conditionals/src/lesson7.ts

```diff
@@ -47,17 +77,26 @@ export function convertGpaToLetterGrade(gpa: number): string {
  * @return The factorial of n.
  */
 export function computeFactorial(n: number): number {
-  return 0;
+  if (n == 0 || n == 1) {
+    return 1;
+  } else if (n > 1) {
+    return n * computeFactorial(n - 1);
+  }
+  return n;
 }
-
+/* 4! = 4 x 3 x 2 x 1 = 24*/
```

Comment:
Prolly shouldn't leave this here. Feel free to update the function documentation to include this example if you want, otherwise just delete.